librz/core/cmd_api.c

```diff
@@ -1879,34 +2002,57 @@ RZ_API const RzCmdDescArg *rz_cmd_desc_get_arg(RzCmd *cmd, const RzCmdDesc *cd,
 	return NULL;
 }
 
+static RzCmdDescHelp *mode_cmd_desc_help(RzCmdDescHelp *dst, const RzCmdDescHelp *src, const char *suffix) {
+	dst->summary = rz_str_newf("%s%s", src->summary, suffix);
+	dst->description = src->description;
+	dst->args_str = src->args_str;
+	dst->usage = src->usage;
+	dst->options = src->options;
+	dst->details = src->details;
+	dst->args = src->args;
+	return dst;
+}
+
 static void cmd_foreach_cmdname(RzCmd *cmd, RzCmdDesc *cd, RzCmdForeachNameCb cb, void *user) {
 	if (!cd) {
 		return;
 	}
 
 	void **it_cd;
 	size_t i;
+	char *name;
```

Comment:
Just define these vars inside the `if (cd->d.argv_modes_data.modes & argv_modes[i].mode) {` as they are used only there.